### PR TITLE
Resources module PR

### PR DESCRIPTION
**Pull request by ishranga11**
- Reformatted index.html and index.css
- Added documentation in app.js
- Corrected app.js to follow Questions Wrapper added in models
- Updated gscribe.sql to follow updated sql schema.
- Removed client id in app.js and added a placeholder.

### CODE REVIEW

File: appengine/src/main/resources/assets/app.js [hunk 1]
```diff
@@ -15,119 +15,342 @@
...
+ * This exam table acts as a compact way to view all exams created by user showing minimal info i.e. exam metadata
+ * The table is filled with exam metadata of all exams created by user
+ * @param data ( exam metadata list response object )
+ */
+function fillExamsTable(data) {
+
+    $('#examsList').empty();
+    let exams = data.examsList;
+    for (var i = 0; i < exams.length; i++) {
+        var examRow = '<tr>' +
```
Comment (dhruvpancholi) [inline appengine/src/main/resources/assets/app.js]: Maybe use jQuery to create this object.

File: appengine/src/main/resources/assets/app.js [hunk 2]
```diff
@@ -15,119 +15,342 @@
...
+}
+
+/**
+ * Called to fill exam metadata in the exam modal
+ * A card is made listing all the exam metadata information
+ * And finally append the card in the exam modal
+ * @param data ( exam metadata )
+ */
+function fillMetadata(data) {
+    let adder = '<div class="card">\n' +
```
Comment (dhruvpancholi) [inline appengine/src/main/resources/assets/app.js]: Use jQuery.

File: appengine/src/main/resources/assets/app.js [hunk 3]
```diff
@@ -15,119 +15,342 @@
...
+}
+
+/**
+ * Called to fill MCQ question into the exam modal
+ * A card is made for the question listing the question fields
+ * Finally the question is appended to the exam modal
+ * @param question
+ */
+function fillMCQ(question) {
+    let adder = '<div class="card">\n' +
```
Comment (dhruvpancholi) [inline appengine/src/main/resources/assets/app.js]: Use jQuery.

File: appengine/src/main/resources/assets/app.js [hunk 4]
```diff
@@ -15,119 +15,342 @@
...
+}
+
+/**
+ * Called to fill SUBJECTIVE question into the exam modal
+ * A card is made for the question listing the question fields
+ * Finally the question is appended to the exam modal
+ * @param question
+ */
+function fillSubjective(question) {
+    let adder = '<div class="card">\n' +
```
Comment (dhruvpancholi) [inline appengine/src/main/resources/assets/app.js]: Use jQuery.

File: appengine/src/main/resources/assets/app.js (hunk 1 above)
Comment (ishranga11) [reply appengine/src/main/resources/assets/app.js]: Done

File: appengine/src/main/resources/assets/app.js (hunk 2 above)
Comment (ishranga11) [reply appengine/src/main/resources/assets/app.js]: Done

File: appengine/src/main/resources/assets/app.js (hunk 3 above)
Comment (ishranga11) [reply appengine/src/main/resources/assets/app.js]: Done

File: appengine/src/main/resources/assets/app.js (hunk 4 above)
Comment (ishranga11) [reply appengine/src/main/resources/assets/app.js]: Done

**Review by dhruvpancholi — APPROVED** [review]

**Pull request merged**